src/main/resources/userEmailTemplate-feedbackSession.html

```diff
@@ -4,9 +4,58 @@
 
 <p>
     The following feedback session ${status}.
-    <br>&nbsp;&nbsp; Course: [${courseId}]${courseName} 
-    <br>&nbsp;&nbsp; Feedback Session Name: ${feedbackSessionName}
-    <br>&nbsp;&nbsp; Deadline: ${deadline}
+    <div>
+        <table style="max-width:600px;border:1px solid black;border-collapse:collapse;margin-left:10px;">
+            <tr>
+                <th colspan="2" style="padding:10px;text-align:center;font-size:120%;">
+                    Feedback Session Details
+                </th>
+            </tr>
+            <tr>
+                <td style="padding:10px;border:1px solid black;">
+                    <strong>
+                        Course:
+                    </strong>
+                </td>
+                <td style="padding:10px;border:1px solid black;">
+                    [${courseId}] ${courseName} 
+                </td>
+            </tr>
+
+            <tr>
+                <td style="padding:10px;border:1px solid black;">
+                    <strong>
+                        Feedback Session Name:
+                    </strong>
+                </td>
+                <td style="padding:10px;border:1px solid black;">
+                    ${feedbackSessionName}
+                </td>
+            </tr>
+
+            <tr>
+                <td style="padding:10px;border:1px solid black;">
+                    <strong>
+                        Deadline:
+                    </strong>
+                </td>
+                <td style="padding:10px;border:1px solid black;">
+                    ${deadline}
+                </td>
+            </tr>
+
+            <tr>
+                <td style="padding:10px;border:1px solid black;">
+                    <strong>
+                        Session Instructions:
+                    </strong>
+                </td>
+                <td style="padding:10px;border:1px solid black;">
+                    ${sessionInstructions}
+                </td>
+            </tr>
+        </table>
```

Comment:
We could also, alternatively, get rid of the table header (**Feedback Session Details**) altogether. The email would then look like this(no borders):

<img width="1280" alt="screen shot 2017-04-16 at 6 00 40 pm" src="https://cloud.githubusercontent.com/assets/18088721/25071202/a9e7d97e-22ce-11e7-9521-1c7be7aa9301.png">